git-ftp

```diff
@@ -50,7 +50,7 @@ CURRENT_BRANCH=""
 SCOPE=""
 KEYCHAIN_USER=""
 KEYCHAIN_HOST=""
-DEPLOYED_SHA1_FILE=".git-ftp.log"
+DEPLOYED_SHA1_FILE=".git-ftp-log"
```

Comment:
Sorry for the `DEPLOYED_SHA1_FILE=".git-ftp-log"` typo.
I tried to let `set_deployed_sha1_file` set the name of the file, but if `DEPLOYED_SHA1_FILE` is empty then, it is not set by `set_deployed_sha1_file `unless the git configuration file has the entry `deployedsha1file`.
I tried this
```
set_deployed_sha1_file() {
        DEPLOYED_SHA1_FILE=".git-ftp.log"
        DEPLOYED_SHA1_FILE="$(get_config deployedsha1file $DEPLOYED_SHA1_FILE)"
        [ $DEPLOYED_SHA1_HIDDEN -eq 1 ] && DEPLOYED_SHA1_FILE="__git-ftp.log"
}
```
where `$DEPLOYED_SHA1_HIDDEN ` is set to 1 if one wants no-hidden file. 
What might be problematic is that then one has to change all hidden files, .git-ftp-ignore etc.  
If you think it is worthwhile I will do another pull request with this and also the save_credential option.